WordPress/src/main/res/values/strings.xml

```diff
@@ -2132,6 +2132,7 @@
     <string name="plugin_detail_banner_desc">plugin banner</string>
     <string name="plugin_detail_logo_desc">plugin logo</string>
     <string name="post_cardview_featured_desc">featured image</string>
+    <string name="post_settings_featured_desc">featured image</string>
```

Comment:
Unless we expect the translations to be different, we should have a string for `featured image` and either use it directly or reference it from these two string resources: `post_cardview_featured_desc` & `post_settings_featured_desc`. From their names, I don't think the translators would interpret them any differently.

Reply:
Good point:)